sdk/test-utils/recorder/test/utils.spec.ts

```diff
@@ -230,4 +228,28 @@ ultramarine.com/url/PUBLIC
       });
     });
   });
+
+  describe("set environment variables", () => {
+    it("should not fail if the dictionary is empty", () => {
+      env.SECRET = "SECRET";
+      const replaceableVariables = {};
+
+      setEnvironmentVariables(replaceableVariables);
```

Comment:
Once setEnvironmentVariables has the environment object parametrized, this unit test won't need to change the global environment object!

Reply:
Updated, but still changing the global environment object to actually test the feature.